tm_examples/detection_map.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+from torchmetrics import MAP
+
+preds = [
+    dict(
+        boxes=torch.Tensor([[258.0, 41.0, 606.0, 285.0]]),
+        scores=torch.Tensor([0.536]),
```

Comment:
please correct if wrong
```suggestion
        # The scores keyword should contain an [N,] tensor where
        # each element is confidence score between 0 and 1
        scores=torch.Tensor([0.536]),
```